src/NClient.Tools/NClient.DotNetTool/Options/ClientOptions.cs

```diff
@@ -0,0 +1,36 @@
+using CommandLine;
+using Microsoft.Extensions.Logging;
+
+namespace NClient.DotNetTool.Options
+{
+    // ReSharper disable once ClassNeverInstantiated.Global
+    [Verb("client", HelpText = "Client interface for OpenAPI")]
```

Comment:
So...  donet nclient client?

Reply:
I can make this method default, then there will be two options:
```C#
dotnet nclient client generate ...
dotnet nclient generate ...
```

Reply:
Nope. Disagree. Verb - is a verb. Client - is not the verb. We must rename this. By default or not - it doesn't matter.

Reply:
Using a noun + verb is a common practice. As an example, I can give a link to the `dotnet ef` documentation:
https://docs.microsoft.com/en-us/ef/core/cli/dotnet#dotnet-ef-database-drop

Reply:
In EF docs I see prefix (dotnet ef) noun and verb.
dotnet nclient - is prefix. Ok.

client generate - is noun and verb. Ok.

just "generate"... Doubtful.

Reply:
Why doubtful? The `generate` verb adds readability to the command, in addition, other verbs will appear later, for example, `validate`

Reply:
I mean this is shouldn't be default. The explicit is better in that case, but dotnet nclient client - is tautology. Can you suggest another naming convention?

Reply:
Done